scripts/package.json

```diff
@@ -9,7 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "nopt": "^3.0.2",
-    "swagger-parser": "^10.0.3"
+    "@apidevtools/swagger-parser": "^10.1.0",
```

Comment:
this thing seems to work on swagger 2 as well? let's land it separately if so.

Reply:
It complains in the Identity Service API about the `3xx` and `4xx` responses status codes. I'm not sure what other syntax could be used for the same thing that would be considered valid swagger2.

Reply:
Nxx notation seems to [only be available in OpenAPI 3](https://swagger.io/docs/specification/describing-responses/#status-codes) - cf. the "HTTP Status Codes" section in the [guide for Swagger 2](https://swagger.io/docs/specification/2-0/describing-responses/).

Reply:
...however, we do (or at least did) use this notation as our custom extension, as documented in [openapi_extensions.md](https://github.com/matrix-org/matrix-spec/blob/main/openapi_extensions.md)

Reply:
If it's a custom extension, shouldn't it start with `x-`? Maybe the validator won't complain then.

Reply:
I am marking this as resolved because updating this dependency right now complains about errors that seem to be gone after the upgrade.